trackers/javascript-tracker/test/functional/cookies.test.ts

```diff
@@ -2,23 +2,24 @@ describe('Tracker created domain cookies', () => {
   it('contain the expected cookie names', async () => {
     await browser.url('/cookies.html');
     await browser.waitUntil(async () => (await $('#init').getText()) === 'true', {
-      timeout: 5000,
+      timeout: 10000,
       timeoutMsg: 'expected init after 5s',
-      interval: 250,
+      interval: 5000,
```

Comment:
Not sure why we need to increase the interval this much?

Reply:
Good catch, that was a relict of my experiments, have now removed.